package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "telemetry-events",
-  "version": "1.0.1",
+  "version": "1.1.0",
```

Comment:
I don't think dependency's version bump should be relevant. I reread this: http://semver.org/